### ISSUE TITLE
Alpine binaries

### ISSUE THREAD

**Issue by skandragon** [kubescape/kubescape#1447]
## Overview
Alpine is a popular OS for containers due to its size.  It would be good to have an Alpine build.

## Problem
The ubuntu image won't run in Alpine (at least :3 and :latest) due to missing shared libraries.

## Solution
Build for alpine.  :)

## Alternatives
If the ubuntu image were build statically, it would also likely run on Alpine.

## Additional context
```
/ # /usr/local/bin/kubescape
/bin/sh: /usr/local/bin/kubescape: not found

/ # ldd /usr/local/bin/kubescape
	/lib64/ld-linux-x86-64.so.2 (0x7efd66da7000)
Error loading shared library libresolv.so.2: No such file or directory (needed by /usr/local/bin/kubescape)
	libpthread.so.0 => /lib64/ld-linux-x86-64.so.2 (0x7efd66da7000)
	libdl.so.2 => /lib64/ld-linux-x86-64.so.2 (0x7efd66da7000)
	libc.so.6 => /lib64/ld-linux-x86-64.so.2 (0x7efd66da7000)
Error relocating /usr/local/bin/kubescape: __fprintf_chk: symbol not found
Error relocating /usr/local/bin/kubescape: __memcpy_chk: symbol not found
Error relocating /usr/local/bin/kubescape: __vfprintf_chk: symbol not found
Error relocating /usr/local/bin/kubescape: __vsnprintf_chk: symbol not found
Error relocating /usr/local/bin/kubescape: __strncpy_chk: symbol not found
Error relocating /usr/local/bin/kubescape: __snprintf_chk: symbol not found
Error relocating /usr/local/bin/kubescape: __printf_chk: symbol not found
Error relocating /usr/local/bin/kubescape: __res_search: symbol not found
```

**Comment by matthyx** [comment]
Thanks for the suggestion... I've planned to make all future releases of kubescape statically linked, without cgo, so running them on alpine shouldn't be an issue.
Stay tuned!

**Comment by matthyx** [comment]
fixed since v3.0.3
